react/features/virtual-background/reducer.js

```diff
@@ -0,0 +1,27 @@
+import { ReducerRegistry } from '../base/redux';
+
+import { SET_VIRTUAL_BACKGROUND } from './actionTypes';
+
+/**
+ * Initial state.
+ */
+const DEFAULT_STATE = {
```

Comment:
Both of these are falsey when undefined, you don't need to define them explicitly, do you?

Reply:
If I don't define them -> I get this console error: "The initial state may not be undefined."

Reply:
Doesn't this do it?

```
ReducerRegistry.register('features/virtual-background', (state = {}, action) => {
```